docker-compose.yml

```diff
@@ -359,7 +359,10 @@ services:
       - DJANGO_SETTINGS_MODULE=koku.settings
       - PROMETHEUS_MULTIPROC_DIR=/tmp
       - SCHEDULE_REPORT_CHECKS=True
-      - REPORT_DOWNLOAD_SCHEDULE="0 4,16 * * *"
+      - REPORT_DOWNLOAD_SCHEDULE_GCP="0 4,16 * * *"
+      - REPORT_DOWNLOAD_SCHEDULE_AWS="0 4,16 * * *"
+      - REPORT_DOWNLOAD_SCHEDULE_AZURE="0 4,16 * * *"
+      - REPORT_DOWNLOAD_SCHEDULE_OCI="0 4,16 * * *"
```

Comment:
We _can_ change these but there is probably no point in the docker-compose file since this stuff doesn't run locally. At the same time I just set these as our current default since they would all be configurable through app-interface.